domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -63,10 +63,6 @@ enum class ReviewScheduleStatus(
   fun isExemptionOrExclusion(): Boolean {
     return isExemption || isExclusion
   }
-
-  fun includeExceptionOnSummary(): Boolean {
```

Comment:
As above, not strictly part of the bug fix, but as part of diagnosing it I ended up refactoring elements of `SessionSummaryService` and I've implemented this method as a private extension function in `SessionSummaryService` because that is closer and more relevant to where it is used